pkg/volume/flexvolume/util.go

```diff
@@ -142,7 +142,7 @@ func prepareForMount(mounter mount.Interface, deviceMountPath string) (bool, err
 func doMount(mounter mount.Interface, devicePath, deviceMountPath, fsType string, options []string) error {
 	err := mounter.MountSensitiveWithoutSystemd(devicePath, deviceMountPath, fsType, options, nil)
 	if err != nil {
-		klog.Errorf("Failed to mount the volume at %s, device: %s, error: %s", deviceMountPath, devicePath, err.Error())
+		klog.ErrorS(err, "Failed to mount the volume at device", "deviceMountPath", deviceMountPath, "devicePath", devicePath)
```

Comment:
How would the error be rendered in this case? Do you have it handy?

Asking because shouldn't `err` and `"Failed to mount the volume at device"` be in the reverse order? Shouldn't `err` be the second param and the string the first one?